modules/pages/client/components/Contribute.component.js

```diff
@@ -59,6 +59,12 @@ export default function Contribute() {
                 </a>{' '}
                 to get an idea of what’s in the making and what you can do to
                 help out.
+                You could help Trustroots out with as little effort as{' '}
+                <a href="https://team.trustroots.org/Chat.html">
+                  donating your photo's
+                </a>{' '}
+                to use for our website or create content. If you're
+                a (semi-)professional photographer or have any questions you can contact us trough share@trustroots.org.
```

Comment:
I think it would be best to just link to contact form here (`<a href="/contact">contact us</a>`) because we have a bit of tendency not to follow that email all the time, but we do follow support queue. Folks can then forward to right place from there if need.

Reply:
Okay, that's also fine. I thought about share because Jans and Shawn also gets those emails and I asked them to keep an eye on it. Whichever you prefer is fine :)